students/hellosputnik/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+)
+
+type Problem struct {
+	question string
+	answer   string
+}
+
+type Quiz struct {
+	problems []Problem
+	score    int
+}
+
+type Settings struct {
+	filename  *string
+	timeLimit *int
+}
+
+func main() {
+	quiz := Quiz{}
+	settings := Settings{}
+
+	settings.filename = flag.String("csv", "problems.csv", "a csv file in the format of 'question,answer'")
+	settings.timeLimit = flag.Int("limit", 30, "the time limit for the quiz in seconds")
+
+	// Get the flags (if any).
+	flag.Parse()
+
+	// Create a file handle for the file.
+	file, err := os.Open(*settings.filename)
+
+	// If there was an error opening the file, exit.
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
Probably a personal preference, but for code like `os.Open(); ... if err ... defer file.Close()` I prefer to group it together without spaces. For me it just makes it easier when things are "grouped" and it makes it obvious that I didn't forget to defer closing the file.